lib/rest-router.js

```diff
@@ -0,0 +1,37 @@
+module.exports = RestRouter;
+
+var url = require('url');
+var Trie = require('./trie');
+
+function RestRouter(options) {
+  this.trie = new Trie();
```

Comment:
Do you expect router consumers to access `router.trie` directly? I think this property should be treated as private, in which case it should be called `_trie`.